flink-connector-aws/flink-connector-aws-kinesis-streams/pom.xml

```diff
@@ -102,6 +115,15 @@ under the License.
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <!-- the kinesis aggregator dependency since it is not available in maven central -->
+            <!-- look into issue https://github.com/awslabs/kinesis-aggregation/issues/120 -->
+            <groupId>com.github.awslabs.kinesis-aggregation</groupId>
+            <artifactId>amazon-kinesis-aggregator</artifactId>
```

Comment:
This has Apache 2.0 licence, all good so
https://github.com/awslabs/kinesis-aggregation